incubator/jasperserver/templates/jasperserver-pvc.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.persistence.enabled -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}-jasperserver
```

Comment:
But the Dockerfile itself has not `/bitnami/tomcat` volume exported.
I'm going to  choose the second option.